e2e/cypress/support/commands.ts

```diff
@@ -119,3 +119,45 @@ function LoginHelper(username: string) {
     cy.wait('@getAuth');
   });
 }
+
+Cypress.Commands.add('cleanupUsersByNamePrefix', (idToken: string, namePrefix: string) => {
+  cy.request({
+    auth: { bearer: idToken },
+    method: 'GET',
+    url: '/api/v0/auth/users',
+    failOnStatusCode: false
+  }).then((resp) => {
+    const body = resp.body;
+    for (const user of body.users) {
+      if (user.name.startsWith(namePrefix)) {
+        cy.request({
+          auth: { bearer: idToken },
+          method: 'DELETE',
+          url: `/api/v0/auth/users/${user.username}`,
+          failOnStatusCode: false
```

Comment:
Can we please not copy this around as if it was part of the furniture? I don't think we should ignore bad status codes here, should we? 😃

Reply:
Wasn't thoughtless copy paste. The reason we are ignoring it is because we don't wanna fail on 404.

Reply:
This first reads the users via the api and then filters by prefix. I don't see how this could get a 404